doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                <br/>
+                David can do this by branching from the trunk, making changes to his branch and testing them.
+                <br/>
+                David can then either delete the branch if the results are not improved or when he is happy that the changes are suitable he can merge his branch changes into the up to date trunk.
+            </div>
+        </ul>
+        </p>
+        <h1><a id="S4">General configuration</a></h1>
+        <p>
+        <ul>
+            <div class="page-header">
+                <h3>Editor Configuration</h3>
+            </div>
+            <div class="well">
+            <br/>
+            Configure an editor for Subversion
+            <br/>
+            Subversion will open an editor when we try to create a branch or commit changes
+            to the repository as it generally requires a commit log message to accompany 
+            the change.
+            <br/>
+            The default editor is vi (or gedit when running the Fcm GUI, the Fcm GUI will 
+            not be covered in this tutorial).
+            <br/>
+            To change the default editor set the environment variable SVN_EDITOR
+            <br/>
+            Note that the editor must be able to run in the foreground.
+            <br/>
+            For example, we can add one of the followings in our ~/.kshrc (ksh) or 
+            ~/.bashrc (bash):
+            <br/>
+            # GVim
+            <br/>
+            <code>export SVN_EDITOR='gvim -f'</code>
+            <br/>
+            # Emacs
+            <br/>
+            <code>export SVN_EDITOR=emacs</code>
+            <br/>
+            # gedit
+            <br/>
+            <code>export SVN_EDITOR=gedit</code>
+            <br/>
+            # Kate
+            <br/>
+            <code>export SVN_EDITOR=kate</code>
+            <br/>
+            The editor-cmd option is in the [helpers] section of the user's 
+            ~/.subversion/config file.
+            <br/>
+            For further information on this please see the file.
+            <br/>
+        </ul>
+        </p>
+        <h1><a id="S5">Setting-up your own repository</a></h1>
+        <ul>
+            <div class="well">
+                <h5>General</h5>
+                Always consider how large it is thought that the repository will grow to.
+                <br/>
+                With this in mind carefully where the repository is to be located. For Example:
+                <br/>
+                <ul>
+                    <li>Only locally as it is only ever for personal use and it is small.</li>
+                    <li>On a shared storage area for the group to all working on.</li>
+                    <li>As part of a Fcm project area as all need to be able to access it, note the 
+                        repository can have limited commit access which is done at set-up and can be 
+                        altered as appropriate via the Trac pages (see the Specific 
```

Comment:
@matthewrmshin - is this true or is it as a result of the [optional] commit hooks interacting with some setting in Trac?